src/components/Kv/KvRadio.vue

```diff
@@ -1,25 +1,29 @@
 <template>
-	<div class="styled-radio">
+	<label class="kv-radio">
```

Comment:
@ryan-ludwig we need to bring back the individual identifier on each individual radio control and label combo. This used to manifest like ":label-set="`${name}-radio-all`"" <- taken from autolend settings.

There may be a way to make this automatic based on the value but it's probably better to just have a prop that can then be applied to both the input and label. ie. `radioId = "group"` then the label has `id="label-group` and the input has `id="input-group"`

Reply:
@mcstover Is this for automation or some other purpose? Perhaps we can add the id to the outer element like `<kv-radio id="my-id">` and then automation could do `querySelector('#my-id label')`. Aiming to keep the API of the components themselves as simple and HTML-like as possible.

Reply:
For accessibility, labels should have a `for` attribute that matches the `id` attribute of the input element that it is a label for. It's best pract to do this even when nesting the input within the label.

Reply:
@ryan-ludwig Yes we can try that approach first.